### PR TITLE
add getMultipleAccountsInfo which uses the getMultipleAccounts RPC method

### PR DESCRIPTION
**Pull request by tomjohn1028**
#### Problem
Web3.js is missing the [`getMultipleAccounts`](https://docs.solana.com/developing/clients/jsonrpc-api#getmultipleaccounts) RPC method

#### Summary of Changes
Add `getMultipleAccountsInfo` to the Connection class

Fixes #

### CODE REVIEW

File: web3.js/src/connection.ts [hunk 1]
```diff
@@ -2407,6 +2407,32 @@ export class Connection {
...
 
+  /**
+   * Fetch all the account info for multiple accounts specified by an array of public keys
+   */
+  async getMultipleAccountsInfo(
+    publicKeys: PublicKey[],
+    commitment?: Commitment,
+  ): Promise<AccountInfo<Buffer>[] | null> {
+    const keys = publicKeys.map(key => key.toBase58());
+    try {
```
Comment (oJshua) [inline web3.js/src/connection.ts:2418]: This outer try block isn't necessary. (based on convention established by other calls and redundant throw if 'error' exists in res)

**Review by oJshua — CHANGES_REQUESTED** [review]
Thanks for adding this! Just one minor change and would you mind adding a quick test?

**Comment by tomjohn1028** [comment]
@oJshua I made the requested changes, but it seems the `npm run test:live` command fails. I'm not up to speed on the different test commands/environments. Care to help me bring this home? Or provide some more context so I can understand what's missing?

**Comment by oJshua** [comment]
Thanks @tomjohn1028 for your effort on this. I simply added a couple airdrops using the helpers.

**Review by oJshua — APPROVED** [review]

**Pull request merged**